documentation/antora.yml

```
@@ -15,6 +15,7 @@ asciidoc:
     modules: '../../modules'
     mysql-version: '8.0'
     strimzi-version: '0.13.0'
+    community: 
```

```suggestion
    community: true
```

In order for the Antora build to actually recognize this attribute, the yaml file must specify an accompanying value in order for it to be injected into the build definitions; otherwise its omitted entirely and omits any section that is scoped by `ifdef::community[]`.